src/AssetRegistry.jl

```diff
@@ -27,8 +27,7 @@ julia> key = AssetRegistry.register("/Users/ranjan/.julia/v0.6/Tachyons/assets/t
 """
 function register(path; registry_file = joinpath(homedir(), ".jlassetregistry.json"))
     target = normpath(abspath(expanduser(path)))
-    (isfile(target) || isdir(target)) ||
-                    error("Asset not found")
+    (isfile(target) || isdir(target)) || error("Asset not found")
```

Comment:
Not blocking, but I think this would look much better as
```suggestion
    if !ispath(target)
        error("Asset not found")
    end
```